hbase-balancer/src/main/java/org/apache/hadoop/hbase/MetaTableAccessor.java

```diff
@@ -193,8 +133,8 @@
    * @param connection connection we're using
    * @param visitor Visitor invoked against each row in regions family.
    */
-  public static void fullScanRegions(Connection connection, final Visitor visitor)
-      throws IOException {
+  public static void fullScanRegions(Connection connection,
```

Comment:
Should this method be in ClientMTA altogether? Or you leave it here just to minimize code churn?